es-runtime/package.json

```diff
@@ -1,43 +1,43 @@
 {
   "name": "@moesol/es-runtime",
   "version": "2.21.0-SNAPSHOT.0",
-  "module": "./build/es-api.js",
+  "type": "module",
```

Comment:
"type": "module" => makes it easier to build es-runtime w/ multiple entrypoints and imports